### DISCUSSION TITLE
Certain file names are not processed correctly

### DISCUSSION THREAD

**Discussion by madiganz** [community/community#26966] [Actions]
<p>For some reason, when I use certain files names, they appear as starred in every step. Is this because GitHub thinks that it is a bad word?</p>
<p><img src="https://user-images.githubusercontent.com/18340029/181100987-f37432c6-5b28-4e2f-b6bf-92e12bb824fb.png" alt="GitHubActions.PNG" data-base62-sha1="ntXGWmVvPXU3Tg1NBCjgU1oEI3T" width="388" height="136"></p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/madiganz">@madiganz</a> , </p>
<p>I guess this *** is caused by masked values. Do you use any secrets variables in your workflow which value macthes the masked string in file name before _ builder ? </p>
<p>For example, if you have a secret variable names ‘env’ which value is ‘product’ ,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/780925ce895e011b98c0f2a3307f66336b9340ae.png" data-download-href="/uploads/short-url/h7STvqOgJmcWJUVsRMcNcpLoWOq.png?dl=1" title="secrets.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181100989-ea855f42-180d-4995-b646-0a950b8fc72b.png" alt="secrets.png" data-base62-sha1="h7STvqOgJmcWJUVsRMcNcpLoWOq" width="690" height="308" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/7/780925ce895e011b98c0f2a3307f66336b9340ae_2_690x308.png, https://aws1.discourse-cdn.com/github/original/2X/7/780925ce895e011b98c0f2a3307f66336b9340ae.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/7/780925ce895e011b98c0f2a3307f66336b9340ae.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/780925ce895e011b98c0f2a3307f66336b9340ae_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">secrets.png</span><span class="informations">1011×452 17.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>and in your workflow you used secrets env in an step environment variable. </p>
<pre><code>jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: print secrets
      run: |
         echo 'product_build'
      env:
        ENV: ${{secrets.env}}
</code></pre>
<p>Then in workflow run logs, all the string “product” will be insteaded by “***”</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/dc2c33af3931431fb78be762cf9f1e41e8883783.png" data-download-href="/uploads/short-url/vpJKe9jnpvZsYpnInu8VY7Yxm7x.png?dl=1" title="mask.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181100991-d8e5a111-885a-4b89-97f0-b6d49dfacbf5.png" alt="mask.png" data-base62-sha1="vpJKe9jnpvZsYpnInu8VY7Yxm7x" width="690" height="401" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/dc2c33af3931431fb78be762cf9f1e41e8883783_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">mask.png</span><span class="informations">691×402 8.57 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Or do you use <a href="https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#masking-a-value-in-log" rel="nofollow noopener">add-mask command</a> ? </p>
<p>If this could not match your scenario, could you please tell me your complete file name? </p>

**Comment by madiganz** [comment]
<p>Hi <a class="mention" href="/yanjingzhu">@yanjingzhu</a>,</p>
<p>I was wondering if that was the problem. Yes, in this case there is a secret that matches that part of the file name. So it shows as a masked string, but it shouldn’t cause any processing problems right?</p>

**Comment by Yanjingzhu** [comment]
<p>No, it is just a display method to protect your secrets in workflow run logs. This masked string will not cause any processing problem.  </p>
